src/main/groovy/edu/oregonstate/mist/locations/core/ExtensionLocation.groovy

```diff
@@ -0,0 +1,34 @@
+package edu.oregonstate.mist.locations.core
+
+class ExtensionLocation {
+    private static final String GEO_LOCATION_REGEX = "-?\\d+(\\.\\d+)?"
```

Comment:
i left this comment because both regular expressions are identical:
```
    public static final String VALID_LAT_LONG = "-?\\d+(\\.\\d+)?"
```

```
    private static final String GEO_LOCATION_REGEX = "-?\\d+(\\.\\d+)?"
```